sources/packages/api/src/database/entities/offering-status.type.ts

```diff
@@ -3,14 +3,12 @@
  */
 export enum OfferingStatus {
   Approved = "Approved",
-  // This status will be changed to CreationPending.
-  Pending = "Pending",
-  // This status will be changed to CreationDeclined.
-  Declined = "Declined",
-  //This status will be changed to ChangeUnderReview.
-  UnderReview = "Under review",
-  // This status will be changed to ChangeAwaitingApproval.
-  AwaitingApproval = "Awaiting approval",
+  // Status with respect to offering creation.
```

Comment:
Please change the comments to the format VS Code will use and please add some comments from a business perspective to these enum options.